userspace/falco/CMakeLists.txt

```diff
@@ -1,12 +1,12 @@
-include_directories(${PROJECT_SOURCE_DIR}/../sysdig/userspace/libsinsp/third-party/jsoncpp)
+include_directories("${PROJECT_SOURCE_DIR}/../sysdig/userspace/libsinsp/third-party/jsoncpp")
```

Comment:
Would it be better to call this CMake command only once with a longer parameter list?